src/VaultSharp/V1/SecretsEngines/Transit/TransitSecretsEngineProvider.cs

```diff
@@ -106,5 +106,17 @@ public async Task<Secret<EncryptionResponse>> RewrapAsync(string keyName, Rewrap
                 rewrapRequestOptions)
                     .ConfigureAwait(_polymath.VaultClientSettings.ContinueAsyncTasksOnCapturedContext);
         }
+
+        public async Task TrimKeyAsync(string keyName, TrimKeyRequestOptions trimKeyRequestOptions, string mountPoint = null)
```

Comment:
This is integration-testable using program.cs RunTransitSamples method. Feel free to test it locally, if not I'll add the test code before publishing the nuget to verify this.

Reply:
You can go ahead and add the test code.